config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
   get "/healthcheck/live", to: proc { [200, {}, %w[OK]] }
   get "/healthcheck/ready", to: GovukHealthcheck.rack_response
+
+  namespace :v1 do
+    resources :locations
```

Comment:
This gives us [all the routes](https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions), and I think we only need a `GET /locations ` right?